FrontPage/controllers/PostTableViewCell.swift

```diff
@@ -0,0 +1,32 @@
+import UIKit
+import Apollo
+
+class PostTableViewCell: UITableViewCell {
+  var taskId: String?
+  
+  @IBOutlet weak var titleLabel: UILabel!
+  @IBOutlet weak var bylineLabel: UILabel!
+  @IBOutlet weak var votesLabel: UILabel!
+  @IBOutlet weak var titleAdd: UITextField!
+  @IBOutlet weak var descriptionAdd: UITextField!
+  
+  func configure(with task: TaskFields) {
+    taskId = task.id
+    titleLabel?.text = task.title
+    bylineLabel?.text = task.description
+    votesLabel?.text = "String for now"
```

Comment:
I think we can remove this now.

Reply:
only votes could be removed as the rest assigns fields displayed to correct values.